setup/productionize/persistence/orca-sql.md

```diff
@@ -84,6 +84,27 @@ executionRepository:
 monitor:
   activeExecutions:
     redis: false
+
+# Use SQL for Orca's work queue
+# Settings from Netflix and may require adjustment for your environment
+# Only validated with AWS Aurora MySQL 5.7
+# Please PR if you have success with other databases
+keiko:
+  queue:
+    fillExecutorEachCycle: true
+    sql:
+      enabled: true
+    redis:
+      enabled: false
+
+queue:
+  zombieCheck:
+    enabled: true
```

Comment:
[Defaults to `false`](https://github.com/spinnaker/orca/blob/master/orca-queue/src/main/kotlin/com/netflix/spinnaker/orca/q/metrics/ZombieExecutionCheckingAgent.kt#L65) - unsure implications